libs/css-codegen/src/lib/style-utils.spec.ts

```diff
@@ -0,0 +1,25 @@
+import * as utilsToTest from './style-utils';
+
+describe('style-utils', () => {
+  describe('convertValue', () => {
+    it('should not convert a non numeric value', () => {
+      expect(utilsToTest.convertValue('block')).toBe('block');
+    });
+    it('should convert an exponential value', () => {
+      expect(utilsToTest.convertValue('-3.019895302713849e-7px')).toBe('1px');
```

Comment:
why does `-3.019895302713849e-7px` get converted to `1px`? Shouldn't it be `-3px`?

Reply:
"-3.019895302713849e-7" is an exponential number, if we get its value we get 0.999999..
so it is rounded to 1

Reply:
Gotcha! Can we also test different numbers with different digits, fraction and sign values e.g. `-387364.019895302713849e-4` or `387364.0198959e-4`?

TBH I don't know what kind of values we might extract from a SketchApp file 😂

Reply:
Yes of course, I agree with you !
I've used values that are present in the current SketchApp sample files. 
I'll add more test cases, it'd be helpful for other people to understand and it'd cover the code I've written

Reply:
Thank you 👍